auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/EntitlementService.java

```diff
@@ -124,6 +125,10 @@ public boolean awsNativeEnabled(String accountId) {
         return isEntitlementRegistered(accountId, CDP_CB_AWS_NATIVE);
     }
 
+    public boolean awsVariantMigrationEnable(String accountId) {
+        return isEntitlementRegistered(accountId, CDP_CB_AWS_VARIANT_MIGRATION);
```

Comment:
CDP_CB_AWS_NATIVE_MIGRATION?

Reply:
What is the difference? We are migrating the AWS variant to AWS_NATIVE variant. What do we want to mark here?